DeDriftAndResample/DeDriftAndResamplePipeline.sh

```diff
@@ -60,10 +60,9 @@ PARAMETERs are [ ] = optional; < > = user supplied value
        (e.g. 32@59)
    --registration-name=<regname> String corresponding to the MSMAll or other registration sphere name 
        (e.g. \${Subject}.\${Hemisphere}.sphere.\${RegName}.native.surf.gii)
-   --dedrift-reg-files=</Path/to/File/Left.sphere.surf.gii@/Path/to/File/Right.sphere.surf.gii> 
-       Path to the spheres output by the MSMRemoveGroupDrift pipeline or NONE
-   --concat-reg-name=<regname> String corresponding to the output name of the concatenated registration 
-       (i.e. the dedrifted registration)
+   [--dedrift-reg-files=</Path/to/File/Left.sphere.surf.gii@/Path/to/File/Right.sphere.surf.gii>]
```

Comment:
Shouldn't the Usage statement indicate which arguments have default settings (and what those defaults are)?  Otherwise, how are users supposed to know which are optional?